### DISCUSSION TITLE
Switching from mtex 5.1 to 5.9

### DISCUSSION THREAD

**Discussion by IvankaMW** [mtex-toolbox/mtex#1853] [Ask Anything] [answered]
Hi,
I had to switch to a newer mtex (and matlab!) version, 
but now I have some pretty basic issues...
(1) plot(ebsd) error:
"Error using patch
Invalid color name or hexadecimal color code. Valid names include: 'red', 'green', 'blue', 'cyan', 'magenta',
'yellow', 'black', 'white', and 'none'. Valid hexadecimal color codes consist of '#' followed by three or six
hexadecimal digits.
Error in EBSD/plotUnitCells (line 76)
h = optiondraw(patch(obj,'parent',ax),varargin{:});
Error in EBSD/plot (line 115)
    h(k) = plotUnitCells(ebsd.subSet(ind), color,..."

(2) similarly:
"oM.inversePoleFigureDirection = yvector;
colorY=oM.orientation2color(ebsd.orientations);"
Error: "Unrecognized field name "orientation2color"."

What am I missing? Is something wrong with my mtex installation?
Thanks a lot!

**Comment by kilir** [comment, marked as answer]
Hi,
wrt 1) Simply use a valid colorname, those have changes you you may the old ones still in your definition of the crystalSymmetry.
wrt2) What is oM? if it is an orientationColorKey of some sort (e.g. ipfHSVKey or axisAngleColorKey ...) the function should exist. However if your oM is a directionColorKey (e.g. HSVDirectionKey) it does not since it takes direction as input via direction2color.

Hope this helps.
Cheers,
Rüdiger

**Reply by IvankaMW** [reply]
Thanks a lot, Rüdiger!
That was a silly simple fix.
All the best,
Ivanka
